packages/vm/package.json

```diff
@@ -43,13 +43,15 @@
     "lint": "../../config/cli/lint.sh",
     "lint:fix": "../../config/cli/lint-fix.sh",
     "formatTest": "node ./scripts/formatTest",
-    "tsc": "../../config/cli/ts-compile.sh"
+    "tsc": "../../config/cli/ts-compile.sh",
+    "examples": "ts-node ../../scripts/examples-runner.ts -- vm"
   },
   "dependencies": {
     "@ethereumjs/block": "^3.6.0",
     "@ethereumjs/blockchain": "^5.5.1",
     "@ethereumjs/common": "^2.6.0",
     "@ethereumjs/tx": "^3.4.0",
+    "@ethereumjs/test-cases": "0.0.1",
```

Comment:
We can't have such an addition to the main dependencies.

(or do you want to really deploy these examples into production for people to run them when having the package installed? Then we would need to discuss, that's a big change, no opinion on that yet but sceptical)